packages/core/src/tracing/instrumentation/protocols/httpServer.js

```diff
@@ -203,3 +206,12 @@ function shimEmit(realEmit) {
     });
   };
 }
+function logInstanaHeaders(req) {
```

Comment:
Added as part INSTA-15032, to check we are getting the trace ids from the header. WE can revert back this later